### PR TITLE
14398 Fix issue with nul values

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/entity#14398

*Description of changes:*

-Add some null value checks.  Was preventing saving a draft after a delete/undo owner scenario...

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts [hunk 1]
```diff
@@ -71,6 +71,8 @@ export const useMhrInformation = () => {
 
     getMhrTransferHomeOwnerGroups.value.forEach(ownerGroup => {
       if (ownerGroup.action !== ActionTypes.REMOVED || isDraft) {
+        ownerGroup.interestDenominator = ownerGroup.interestDenominator || 0
```
Comment (dimak1) [inline ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:74]: Won't this affect the field validation? When they are null, the validation kicks in, but if they are 0's what happens?

Comment (ChasBoucher) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:74]: This only occurs on saving after validation.  The API was returning an error if these values were null..

**Review by dimak1 — APPROVED** [review]

**Comment by ChasBoucher** [comment]
Closed because of rebase issue after commiting version update.. Easier to make new pr.. Only 2 lines of code changed...

**Pull request closed without merging**